### PR TITLE
[TOPI] upsample operator 'NCHWinic' format support.

### PR DESCRIPTION
**Pull request by huajsj**
some hardware accelerator ask packed format data like NCHWinic to fit the
hardware resource, here add 'upsample' 'NCHWinic' format support to help
such requirement.

### CODE REVIEW

**Comment by huajsj** [comment]
Hi @Laurawly @Huyuwei , could you help for the review thanks.

**Comment by huajsj** [comment]
Hi @tmoreau89 , this patch related to let VTA to support Yolov3, if you have time could you help for a review too?

**Comment by huajsj** [comment]
Hi @jwfromm, if you have time could you help for a review? thanks.

**Comment by tmoreau89** [comment]
@srkreddy1238 could you help review?

File: topi/python/topi/image/resize.py [hunk 1]
```diff
@@ -18,8 +18,37 @@
...
+    if boxes is None:
+        y = tvm.max(tvm.min(y, image_height - 1), 0)
+        x = tvm.max(tvm.min(x, image_width - 1), 0)
+    if layout == 'NHWC':
+        return data(n, y, x, c).astype('float')
+    if layout == 'NCHW':
+        return data(n, c, y, x).astype('float')
+    if nchw_pack_layout(layout):
+        return data(n, c, y, x, ib, ic).astype('float')
+    # else must be NCHWxc
```
Comment (tmoreau89) [inline topi/python/topi/image/resize.py]: can we assert this

Comment (huajsj) [reply topi/python/topi/image/resize.py]: Hi @tmoreau89 , thanks for the follow up, sure, issue just fixed.

**Comment by tmoreau89** [comment]
Thanks @huajsj @Laurawly @jwfromm the PR has been merged.

**Pull request merged**